Ромахина Галина

GalinaRoma opened the pull request:
[Посмотреть решение](https://rawgit.com/GalinaRoma/markup-task-7/master/index.html)

GalinaRoma committed:
atom done

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

bratva commented on images/play.svg:
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
```
ужас какой-то выдаёт этот сервис. Попробуй сама нарисовать или прогани через svgo

bratva commented on index.css:
```diff
@@ -1 +1,449 @@
+input
+{
+    display: none;
```
Вы все прямо сговорились. Откуда и зачем вы это копируете? Зачем скрывать инпут через display:none?

GalinaRoma replied:
А что плохого в этом методе? Как-то можно лучше? Я так во всех практиках писала, не знала, что это плохо

bratva replied:
Так теряется доступ с клавиатуры и для читалок. Посмотри доклад Вадима, как и что лучше прятать
https://htmlacademy.ru/shorts/12

GalinaRoma replied:
Хорошо, посмотрю

bratva reviewed (commented):
🍅 
Логотип сильно отличается от макета. Вся вёрстка прибита гвоздями. Если текста станет больше, то всё поедет. Нужно исправить.
<img width="356" alt="2017-11-29 11 05 01" src="https://user-images.githubusercontent.com/1937870/33360577-17aa8b76-d4f6-11e7-9f49-237952e44382.png">

GalinaRoma commented:
Логотип, который logo-atom сильно отличается?
А по поводу гвоздей, то есть лучше было всю верстку через margin делать?
Ведь тогда при вставке доп текста тоже всё поедет

bratva commented:
Нет не через margin. 
Просто тебе нужно позиционировать эти элементы относительно контейнера с текстом или контейнеров под этими текстами.

GalinaRoma committed:
Поменяла логотипы и прогнала через svgo;
Изменила у input display: none на visibility: hidden;
Исправила основную часть так, чтобы при добавлении текста ничего не ломалось

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

bratva commented:
![2017-12-03 11-10-41](https://user-images.githubusercontent.com/1937870/33522902-a0269f40-d81a-11e7-9528-724ba3214423.jpg)
🍅

GalinaRoma committed:
Исправила так, чтобы придобавлении текста, ничего не ломалось

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты
